swagger/public_v1/template.yml

```diff
@@ -435,6 +435,66 @@ components:
             $ref: "#/components/schemas/LocationResource"
         jsonapi:
           $ref: "#/components/schemas/JSONAPI"
+    LocationStatus:
+      type: object
+      description: Wrapper object for LocationStatusAttributes
+      properties:
+        id:
+          type: string
+          example: "22329867"
+        type:
+          type: string
+          example: site_statuses
+        attributes:
+          $ref: "#/components/schemas/LocationStatusAttributes"
+    LocationStatusAttributes:
+      type: object
+      description: Fields relating the state of a location
+      properties:
+        status:
+          type: string
+          enum:
+            - discontinued
+            - running
+            - new_status
+            - suspended
+          example: "running"
+        publish:
+          type: string
+          enum:
+            - published
+            - unpublished
+          example: "published"
+        has_vacancies:
+          type: boolean
+          description: "Are there any vacancies for this course"
+          example: true
+        vac_status:
+          type: string
+          description: "What type of vacancies are available"
+          enum:
+           - full_time_vacancies
+           - part_time_vacancies
+           - both_full_time_and_part_time_vacancies
+          example: full_time_vacancies
+        relationships:
+          $ref: "#/components/schemas/LocationStatusRelationships"
+    LocationStatusRelationships:
```

Comment:
What about putting the relationships file into the `component_schemas` dir as a pattern? That's what we did with `CourseAttributes` and `ProviderAttributes`.

Reply:
That's not in master yet and only exists in http://github.com/DFE-Digital/teacher-training-api/pull/1278/

Reply:
That PR is merged now.

Reply:
i've rebased and split out so it aligns with the other files